frontend/src/components/modals/EditResponseModal.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { Modal, Button } from 'react-bootstrap'
+import { useTranslation } from 'react-i18next/'
+
+const EditResponseModal = ({
```

Comment:
Csak egy kérdés: ez miben speciálisabb, mint más modal? Ez mitől lesz EditResponseModal? Ha ugyanaz, mint máshol akkor lehetne egy ResponseModal, ami kellően általános, mint ez is és azt lehetne használni mindenhol.

Reply:
Amíg ott volt a navigálás, két ResponseModal volt. Most már egy lett.